nupic/regions/CLAClassifierRegion.capnp

```diff
@@ -0,0 +1,14 @@
+@0xf091b1bb874049df;
+
+using import "/nupic/bindings/proto/ClaClassifier.capnp".ClaClassifierProto;
```

Comment:
> Please use relative paths for now (there is a bug in pycapnp that makes mixing relative and absolute paths cause problems)

@scottpurdy I saw a discussion between you and @oxtopus about this. Is this a final solution?